packages/hub/src/lib/list-models.ts

```diff
@@ -63,6 +63,7 @@ export async function* listModels<
 			owner?: string;
 			task?: PipelineType;
 			tags?: string[];
+			inferenceProvider?: string;
```

Comment:
You can search for several providers at once
```suggestion
			inferenceProvider?: string[];
```

Reply:
yes it's an `OR`

Reply:
Thank you @frascuchon !